scylla-cdc-printer/Cargo.toml

```diff
@@ -14,3 +14,4 @@ futures = "0.3.17"
 anyhow = "1.0.51"
 async-trait = "0.1.52"
 clap = { version = "3.1.10", features = ["derive"] }
+scylla-cdc-utils = { path = "../scylla-cdc-utils" }
```

Comment:
This should be put into the `dev-dependencies` section, as those utilities are only used in the tests.